lib/src/lib.rs

```diff
@@ -107,6 +107,8 @@ extern crate tracing;
 mod mac;
 
 mod api;
+
+pub use api::opt::Config;
```

Comment:
`Config` is already public. Is there any reason you want to re-export it from the root?

Reply:
```
    let db: Surreal<Any> = Surreal::init();

    let uri = format!("{}://{}", input.payload.protocol, input.payload.uri);
    db.connect((uri, Config::new().capabilities(Capabilities::all())))
        .await
        .map_err(|e| SurrealError::FailedToConnect(e.to_string()))?;
```
I need this when running embed tikv to enable scripting.

Reply:
Yes @0xDjole but [it's already public](https://docs.rs/surrealdb/1.0.0/surrealdb/opt/struct.Config.html). It doesn't have to be in the root of the crate. You can just

```rust
use surrealdb::opt::Config;
```

to bring it into scope.

Reply:
Nvm.